js/data/item_data.js

```diff
@@ -465,43 +465,8 @@ function getTechnoBlast(item) {
 }
 
 function getMultiAttack(item) {
-	switch (item) {
-	case 'Bug Memory':
-		return 'Bug';
-	case 'Dark Memory':
-		return 'Dark';
-	case 'Dragon Memory':
-		return 'Dragon';
-	case 'Electric Memory':
-		return 'Electric';
-	case 'Fairy Memory':
-		return 'Fairy';
-	case 'Fighting Memory':
-		return 'Fighting';
-	case 'Fire Memory':
-		return 'Fire';
-	case 'Flying Memory':
-		return 'Flying';
-	case 'Ghost Memory':
-		return 'Ghost';
-	case 'Grass Memory':
-		return 'Grass';
-	case 'Ground Memory':
-		return 'Ground';
-	case 'Ice Memory':
-		return 'Ice';
-	case 'Poison Memory':
-		return 'Poison';
-	case 'Psychic Memory':
-		return 'Psychic';
-	case 'Rock Memory':
-		return 'Rock';
-	case 'Steel Memory':
-		return 'Steel';
-	case 'Water Memory':
-		return 'Water';
-	default:
-		return '';
+	if (item.indexOf("Memory") !== -1) {
```

Comment:
Plates works the same way, do they not?

Reply:
Nope. Plates' names are like "Meadow Plate", "Flame Plate", while Memories directly point at type such as "Fire Memory", "Grass Memory", etc. so instead of looping through every case we just cut "Fire" from "Fire Memory" and we return it.

Reply:
Ah I forgot the plate naming was different :p